helpers/helpers.sh

```diff
@@ -104,14 +105,12 @@ shell_profile() {
       PROFILE="$HOME/.zshrc"
   fi
 
-  if [ -n "$PROFILE" ]; then
+  if ! [ -z "$PROFILE" ]; then
     export PROFILE
-  elif $(echo $0 | grep bash > /dev/null); then
```

Comment:
@tyler-pave I think removing these lines means this script will be broken if the user is using a bash shell and doesn't yet have a profile file created, since it will be creating/adding the relevant bits to the zsh profile file, which (afaik) isn't used at all by bash. Suggest re-introducing the shell check, or potentially removing support for bash entirely to avoid confusion down the road.

Reply:
yeah i would vote for not supporting bash. I had to change it cause it was not finding the shell with any of the commands that we tried. so i just set it to default to zsh, so devs wouldn't be blocked by this